pageserver/src/bin/pageserver.rs

```diff
@@ -226,6 +228,15 @@ fn start_pageserver(
     launch_ts: &'static LaunchTimestamp,
     conf: &'static PageServerConf,
 ) -> anyhow::Result<()> {
+    // Monotonic time for later calculating startup duration
+    let started_startup_at = Instant::now();
+
+    let startup_checkpoint = move |phase: &str, human_phase: &str| {
+        let secs = started_startup_at.elapsed().as_secs_f64();
+        STARTUP_DURATION.with_label_values(&[phase]).set(secs);
+        info!("{human_phase} ({secs:.3}s since start)")
```

Comment:
Could we at least keep the `elapsed_millis` in these?

Reply:
Done.